### ISSUE TITLE
Does this work with Scrum template?

### ISSUE THREAD

**Issue by DanBirchgrove** [CustomExtensions/OlaTrackingExtension#1]
The references of field names in the description seem to be to the Bug WI in Agile template.

**Comment by Timothy-Dement** [comment]
That's correct, the widget was designed to work with the Agile template. The main difference between the two templates for the purposes of the widget is that the Agile template categorizes bug state by `Active`, `Closed`, `New`, and `Resolved` by default, whereas the Scrum template categorizes bug state by `Approved`, `Committed`, `Done`, `New`, and `Removed` by default. While some of the labels and descriptions do not match the Scrum template, a `Closed` bug in the Agile template and a `Done` bug in the Scrum template are treated the same by the widget.

Thank you for bringing this to my attention, I will plan to update the documentation to reflect the original Agile template design, and will flag this as an `enhancement` for future work.
